pkgs/top-level/all-packages.nix

```diff
@@ -45,7 +49,7 @@ with pkgs;
     by Hydra.
   '';
 
-  tests = recurseIntoAttrs (callPackages ../test {});
+  tests = callPackages ../test {};
```

Comment:
does release.nix recurse these? they are referenced by the constituent jobsets so this would cause evaluation errors if that's not the case

Reply:
Sorry my bad

Reply:
it definitively shouldn't be recursed for nix-env tho, nobody is going to want to install `cc-wrapper-test`. 😄